### PR TITLE
Up to date

### PR DESCRIPTION
**Pull request by Antonio-Laguna**

### CODE REVIEW

File: cssdb.json [hunk 1]
```diff
@@ -128,7 +149,25 @@
...
+    "title": "Container Queries",
+    "description": "New container property and container at rule to make changes depending on the container's size",
+    "specification": "https://drafts.csswg.org/css-contain-3/#container-queries",
+    "stage": 0,
+    "caniuse": "css-container-queries",
+    "example": ".container {\n  contain: layout inline-size;\n}\n\n@container (min-width: 700px) {\n  .container {\n    /* styles applied when a container is at least 700px */\n  }\n}",
+    "docs": {
+      "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries"
+    },
+    "polyfills": [
```
Comment (romainmenke) [inline cssdb.json:167]: Should we consider : https://www.npmjs.com/package/container-query-polyfill

Maybe this is a list where multiple polyfills can be added side by side? I don't know the format that well :)

File: cssdb.json [hunk 2]
```diff
@@ -525,6 +569,7 @@
     "description": "A pseudo-class for matching elements in a selector list",
     "specification": "https://www.w3.org/TR/selectors-4/#matches-pseudo",
     "stage": 2,
+    "caniuse": "css-matches-pseudo",
```
Comment (romainmenke) [inline cssdb.json:572]: Should this be a higher stage?
It's supported in all major browsers.

File: cssdb.json [hunk 3]
```diff
@@ -743,14 +800,17 @@
       }
     },
     "example": "body {\n  background-color: white;\n  color: black;\n}\n\n@media (prefers-color-scheme: dark) {\n  body {\n    background-color: black;\n    color: white;\n  }\n}",
+    "docs": {
+      "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme"
```
Comment (romainmenke) [inline cssdb.json:804]: Maybe also higher stage?

**Review by romainmenke — COMMENTED** [review]
some comments.
Unsure if you also wanted to update the `stage` value for each here?

File: cssdb.json (hunk 1 above)
Comment (Antonio-Laguna) [reply cssdb.json:167]: Great point!

**Review by romainmenke — APPROVED** [review]
LGTM!

**Review by romainmenke — COMMENTED** [review]
Everything looks really nice!

I was wondering if we could define some CSS variables to use as constants for things like spacing.

```css
:root {
  --space-16: 16px;
  ...
}
```

At the moment it feels like there are too many different spacing values and re-using predefined values can help here.

Screenshot 2021-12-30 at 18 51 28

On larger screens were there is more room it feels much better.
So this is mostly on mobile where we get more wrapping.

--------

Currently the browser support is indicated with images and numbers and these have `aria-hidden`.

Can we have the same information in text for screenreader users?

**Comment by Antonio-Laguna** [comment]
@romainmenke thanks for your review!

All makes sense, didn't touch much of the original styles except to transform some unsupported CSS rules but it does make sense. I did some changes to accessibility already and I'm glad you found that out, definitely will fix :)

**Pull request merged**